robot_driver/src/robot_driver.cpp

```diff
@@ -148,6 +136,43 @@ RobotDriver::RobotDriver(ros::NodeHandle nh, int argc, char **argv) {
   double dt = 1.0 / mocap_rate_;
   filter_weight_ = 1.0 - dt / filter_time_constant_;
 
+  // Initialize state and control strucutres
+  initStateControlStructs();
+
+  // Load complementary filter coefficients
+  loadCompFilterParams();
+}
+
+void RobotDriver::initStateEstimator() {
```

Comment:
Ideally, we want the same structure here as the leg controller. Probably need a setGains function inside comp filter class that we can use rather than setting them directly inside robot driver.

Reply:
We could also update LegController if that would make more sense (e.g. LegController::init(gains))?

Reply:
Make sense!